# Measured vehicle speed as reported by vehicle 

std_msgs/Header header

float64 vehicle_speed         # Measured vehicle speed in m/s
bool vehicle_speed_valid      # True if vehicle speed is valid

